src/FormContainer.tsx

```diff
@@ -69,23 +69,30 @@ const makeWrapper = <T extends {}>(config: IFormConfig) => (WrappedComponent: an
             }
         };
 
-        bindInput = (name: keyof T): IBoundInput => ({
-            name,
-            value: this.getValue(name),
-            onChange: this.bindToChangeEvent,
-            onFocus: this.bindToFocusEvent,
-            onBlur: this.bindToBlurEvent,
-            ref: (input: any) => {
-                if (!this.state.inputs[name]) {
-                    this.setState({
-                        inputs: {
-                            ...this.state.inputs,
-                            [name]: input
-                        }
-                    });
+        bindInput = (name: keyof T): IBoundInput => {
+            const inputMethods: IBoundInput = {
+                name,
+                value: this.getValue(name),
+                onChange: this.bindToChangeEvent,
+                onFocus: this.bindToFocusEvent,
+                onBlur: this.bindToBlurEvent,
+            };
+
+            if (React.Component.prototype.render) {
```

Comment:
I'm an idiot. This needs to be `WrappedInput.prototype.render`.

Fixing it now, but also trying to update tests and such.

Reply:
Though that is whether the _WrappedComponent_ has a render method (it always will) not whether the bound input has a render method... 🤔

Reply:
yeah, we cannot assess it in `bindInput`...